controllers/userController.js

```diff
@@ -57,4 +57,64 @@ const emailCheck = async (req, res) => {
   }
 };
 
-module.exports = { signUp, signIn, emailCheck };
+const myPageShow = async (req, res) => {
+  const email = req.user.email;
+  try {
+    const result = await userService.mypage(email);
+    return res.status(200).json({ data: result });
+  } catch (err) {
+    console.log(err);
+    return res.status(err.statusCode || 500).json({ message: err.message });
+  }
```

Comment:
1. 구조분해 할당을 활용해주세요.
2. 구조분해 할당을 하는 코드 또한 try 블록안에 작성해주세요. 
3. 미들웨어에서 토큰을 거쳐 나오는 정보에는 email이 아닌 id가 담기게 해주세요. 
```suggestion
  try {
    const { userId } = req.user
    const result = await userService.getUserInfo(userId);
    return res.status(200).json({ data: result });
  } catch (err) {
    console.log(err);
    return res.status(err.statusCode || 500).json({ message: err.message });
  }
```